docs/swaggerui/swagger.json

```diff
@@ -380,13 +380,14 @@
         }
       }
     },
-    "/api/member/tier": {
+    "/api/member/stats": {
```

Comment:
Did you make sure to rerun the `gendocs` file? This should no longer exist

Reply:
ah, i ran it, but then i didn't run it again